setup.cfg

```diff
@@ -25,16 +25,16 @@ install_requires =
     nipype ~= 1.5.1
     nitime
     nitransforms >= 20.0.0rc3,<20.2
-    niworkflows >= 1.4.0rc0, <1.5
-    nirodents >= 0.2.2
+    niworkflows ~= 1.4.0
+    nirodents >= 0.2.4
     numpy
     pandas
     psutil >= 5.4
     pybids >= 0.11.1
     pyyaml
     requests
     smriprep >= 0.8.0rc0
-    svgutils < 0.3.2
+    svgutils >= 0.3.4
```

Comment:
```suggestion
```

I don't think we call svgutils within fmriprep-rodents, and the pin is done in niworkflows already. Let's simplify